### ISSUE TITLE
Chaining PositionAtSpeed tweens doesn't work as expected

### ISSUE THREAD

**Issue by Romaleks360** [KyryloKuzyk/PrimeTween#71]
```
        Sequence.Create()
            .Chain(Tween.PositionAtSpeed(transform, pos1, _moveSpd, Ease.InCubic))
            .Chain(Tween.PositionAtSpeed(transform, pos2, _moveSpd, Ease.Linear))
            .Chain(Tween.PositionAtSpeed(transform, pos3, _moveSpd, Ease.OutCubic));
```

I expect the following code to move my object sequentially: start -> pos1 -> pos2 -> pos3.
Instead, the object position is reset on each tween: start -> pos1, start -> pos2, start -> pos3.

Using Position instead of PositionAtSpeed works as expected.

**Comment by Romaleks360** [comment]
Providing startValue also works. However, the speed of segments doesn't match. The final max speed is much higher for cubic-eased segments that for linear-eased segment. I had to divide the speed of cubic-eased segments by 3. Here's the final code that gives correct motion:

```
        Sequence.Create()
            .Chain(Tween.PositionAtSpeed(transform, pos1, _moveSpd / 3, Ease.InCubic))
            .Chain(Tween.PositionAtSpeed(transform, pos1, pos2, _moveSpd, Ease.Linear))
            .Chain(Tween.PositionAtSpeed(transform, pos2, targetPos, _moveSpd / 3, Ease.OutCubic));
```

**Comment by KyryloKuzyk** [comment]
> @Romaleks360 Providing startValue also works

Yes, speed-based tweens require the startValue when used inside sequences, please see this doc section: https://github.com/KyryloKuzyk/PrimeTween?tab=readme-ov-file#speed-based-animations

> However, the speed of segments doesn't match

Yes, this is because AtSpeed() methods accept the `averageSpeed` parameter. It's only possible to have the precise speed with Ease.Linear, and all other ease types will modify the actual speed. Like, it's impossible to have easing simultaneously with constant speed. Constant speed == linear motion, i.e. Ease.Linear. So AtSpeed() method can only guarantee the average speed.

> I had to divide the speed of cubic-eased segments by 3

If you're hand-picking the averageSpeed, then probably it's easier to use the regular Tween.Position() in your case.

> Also I can't see TweenSettings overload for PositionAtSpeed :^(

Yes, this is because TweenSetting has only the `duration` property. Of course, I can treat duration as speed, but this will introduce inconsistency as it would not be obvious to the users.
